drivers/client_access_control/app/views/client_access_control/clients/enrollment_details/_enrollment_rollups.haml

```diff
@@ -15,6 +15,9 @@
     %article.rollup.o-dashboard__block.o-dashboard__block--secondary{data: {partial: 'services-all'}}
       %header
         %h1#full_services.h3 Services
+    %article.rollup.o-dashboard__block.o-dashboard__block--secondary{data: {partial: 'custom_services'}}
+      %header
+        %h1#full_services.h3 Custom Services
```

Comment:
This should probably be wrapped in an HMIS check, so its only shown if this warehouse installation has an accompanying HMIS.

```suggestion
    - if HmisEnforcement.hmis_enabled?
      %article.rollup.o-dashboard__block.o-dashboard__block--secondary{data: {partial: 'custom_services'}}
        %header
          %h1#full_services.h3 Custom Services
```

This would still show the "Custom Services" section for a non-HMIS Client in an installation that does support HMIS, but I think that's okay.